plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -758,6 +788,8 @@ public Optional<ConnectorTableExecuteHandle> getTableHandleForExecute(
         switch (procedureId) {
             case OPTIMIZE:
                 return getTableHandleForOptimize(session, tableHandle, executeProperties, retryMode);
+            case DROP_EXTENDED_STATS:
```

Comment:
Should this "DROP EXTENDED STATS" procedure be at least in a separate commit ?

Reply:
it's a logical complement to ANALYZE (otherwise i wouldn't add it).